### DISCUSSION TITLE
Jobs stuck in running for days

### DISCUSSION THREAD

**Discussion by dlen** [apalis-dev/apalis#422] [Q&A] [answered]
Hi,

I have a MySQL backend where I can see jobs stuck in 'Running' state for days but no processing is done. I was planning to add a cron job to set their state back to 'Pending' so they get re-processed but, are there any better options?

**Comment by geofmureithi** [comment]
Let me confirm if this a bug in the processing.

**Comment by geofmureithi** [comment, marked as answer]
This is possibly related to #419 
Please try using the branch `chore/v0.6.0` and get back.
